content/docs/cve/cveIn-PersonExamProcess.md

```diff
@@ -25,26 +26,26 @@ type: book  # Do not modify.
 
 ## Applicant Registration:
 
-1. PRE-REGISTER: Direct applicants to pre-register (and pay online) before the exam by using the link on [HamStudy](https://ham.study). The session listing link may be public or private.  A public link will allow anyone who searched HamStudy to register. A private link where the Do not post on HamStudy is selected during session creation will only those who have the link to access the session listing and register.
+1. PRE-REGISTER: Direct applicants to pre-register (and pay online) before the exam by using the link on [HamStudy](https://ham.study). The session listing may be public or private.  A public session will allow anyone who searched HamStudy to register. A private link will only be viewable to those who have the link to the session listing where the Do not post on HamStudy.org is selected during session creation.  Once a session has been listed publicly, applicants are able to navigate back to it even if the session was edited to hide it on HamStudy. 
```

Comment:
I think something is funny with the private link sentence.

Reply:
I agree.